src/middlewares/schema.spec.ts

```diff
@@ -263,3 +263,21 @@ describe('on schema that contains fields with bail methods', () => {
     expect(context.errors).toHaveLength(1);
   });
 });
+
+it('run checkSchema imperatively', async () => {
+  const req = {
+    body: { foo: 'foo' },
+  };
+  const schema = checkSchema({
+    foo: {
+      exists: true,
+      isLength: {
+        options: {
+          min: 5,
+        },
+      },
+    },
+  });
+
+  expect(schema.run(req)).resolves;
```

Comment:
this needs an `await` or `return` -- otherwise it may fail out of band

Reply:
I forgot the `return`🤦🏻‍♂️